tests/UnitTests/Foundation/NSObject_NSKeyValueOrderedSetAdaptersTests.mm

```diff
@@ -0,0 +1,265 @@
+//******************************************************************************
+//
+// Copyright (c) Microsoft. All rights reserved.
+//
+// This code is licensed under the MIT License (MIT).
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//******************************************************************************
+
+#import <Foundation/Foundation.h>
+#include <TestFramework.h>
+
+@interface TestOrderedSetAdapterObject : NSObject {
+    NSMutableOrderedSet* alreadyMutable;
+    NSOrderedSet* immutable;
+    NSObject* notAnOrderedSet;
```

Comment:
looks like we can't.
@duhowett@microsoft.com  thanks

---
In reply to: [114012028](https://github.com/Microsoft/WinObjC/pull/2647#discussion_r114012028) [](ancestors = 114012028)